README.md

```diff
@@ -207,31 +160,42 @@ Users want to determine several types of the formatting width as following
   + fr-FR
     - 1 heure, 46 minutes et 40 secondes
 
-### Design
+#### Design
 
-Adding the locale in `string` format as a first argument.
+Adding the locale in string format as a first argument, or specifying a ranked list of locales as an array of string locale values.
 
-# API design
+## API design
 
 ``` javascript
-    const formatter =
-        new Intl.DurationFormat('en-US', {
-            fields: [
-                'hour',
-                'minute',
-                'second'
-            ],
-            style: 'short',
-            hideZeroValues: 'none',
-        });
-
-    const duration =
-        Temporal.Duration.from({
-            hours: 2,
-            minutes: 46,
-            seconds: 40
-        });
-
-    console.log(formatter.format(duration));
-    // output: 2 hr 46 min 40 sec
+    new Intl.DurationFormat('en').format(Temporal.Duration.from('PT2H46M40S')); // => 2 hr 46 min 40 sec
+    new Intl.DurationFormat('en', {
+      hours: 'numeric',
+      seconds: 'numeric',
+    }).format(Temporal.Duration.from('PT2H40S')); // => 2:00:40
 ```
+
```

Comment:
There should be a section explaining the motivation and design for the `fractionalDigits` option. Also a section for the `style` option.

Finally, I was hoping to see an "Options Summary" or "Design Summary" section that shows the full list of options and their allowed values. The current layout of the page with multiple, separate "Design" sections is fine, but it makes it visualize every option to understand the big picture of how the API works.  I think both are helpful.

Reply:
The new `fractionalDigits` section with examples is great. But IMHO this page still needs an options summary section which lists all the possible options, their possible values, and a short sentence about each one. Something like the "parameters" section of https://tc39.es/proposal-temporal/docs/zoneddatetime.html#until.

Reply:
@justingrant updated! Please check out the README now and let me know what you think.

Reply:
New content is much clearer and easier to understand. Thanks!